rakefile.netstd.rb

```diff
@@ -48,15 +49,13 @@
 ]
 
 repo = 'FakeItEasy/FakeItEasy'
-teamcity_server_url = 'http://teamcity.codebetter.com/admin/editBuildParams.html?id=buildType:bt929'
 appveyor_server_url = 'https://ci.appveyor.com/project/FakeItEasy/fakeiteasy/settings'

```

Comment:
Oh! This is the URL I've elsewhere in the PR. I guess we'll have to decide one way or the other and then I'll fix them to be consistent.